neurom/apps/morph_check.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2015, Ecole Polytechnique Federale de Lausanne, Blue Brain Project
```

Comment:
I think this should be 
`2015-2021`

Reply:
In all source files this line used. Should we update all other files as well?

Reply:
If I recall correctly (from Andrew like a year ago), I ~~think we should update if the file changed during the year.
For not updated files I think this is fine.

Reply:
@ASVG please clarify on copyright notices